File: Web/descubretusede/model/Horario.php
<?php
     class Horario {
 	public $id_horario;
	public $id_seccion;
	public $id_sala;
	public $dia_clases;
	public $hora_inicio;
	public $hora_termino;
	
	function __construct()
	{
	}
 	
	function init($id_horario,$id_seccion,$id_sala,$dia_clases,$hora_inicio,$hora_termino)
	{
		$this->id_horario=$id_horario;
		$this->id_seccion=$id_seccion;
		$this->id_sala=$id_sala;
		$this->dia_clases=$dia_clases;
		$this->hora_inicio=$hora_inicio;
		$this->hora_termino=$hora_termino;
	}
	
	function __destruct() 
	{
        unset($this);
	}
}
?>

File: android/app/src/main/java/cl/duoc/descubretusede/model/Horario.java
package cl.duoc.descubretusede.model;

import java.sql.Time;

/**
 * Created by kurt on 13-10-2014.
 */
public class Horario {
    private int idHorario;
    private String diaClases;
    private Time horaInicio,horaTermino;
    private Aula aula;
    private int idSeccion;

    public Horario() {
    }

    public int getIdSeccion() {
        return idSeccion;
    }

    public void setIdSeccion(int idSeccion) {
        this.idSeccion = idSeccion;
    }


    public int getIdHorario() {
        return idHorario;
    }

    public void setIdHorario(int idHorario) {
        this.idHorario = idHorario;
    }

    public String getDiaClases() {
        return diaClases;
    }

    public void setDiaClases(String diaClases) {
        this.diaClases = diaClases;
    }

    public Time getHoraInicio() {
        return horaInicio;
    }

    public void setHoraInicio(Time horaInicio) {
        this.horaInicio = horaInicio;
    }

    public Time getHoraTermino() {
        return horaTermino;
    }

    public void setHoraTermino(Time horaTermino) {
        this.horaTermino = horaTermino;
    }

    public Aula getAula() {
        return aula;
    }

    public void setAula(Aula aula) {
        this.aula = aula;
    }
}
